src/core/definitions.jl

```diff
@@ -109,6 +111,9 @@ const INFLOW = "In"
 const SPILLAGE = "Sp"
 const SLACK_UP = "γ⁺"
 const SLACK_DN = "γ⁻"
+const COLD_START = "δ_cold"
```

Comment:
```suggestion
const COLD_START = "start_cold"
```

Keep consistency with other variable names
```